### ISSUE TITLE
open in terminal button not accessible

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#2694]
The `Open in Terminal` button in the files app is not accessible.  A screen reader reads is simply `Menu button collapsed sub menu`. That is, it does not read any title such as 'open in terminal'.
